debian/changelog

```diff
@@ -1,3 +1,9 @@
+tiqit (1.0.6-1) trusty; urgency=low
+
+  * Move to use HTTPS
```

Comment:
We're also releasing Kyle's fix to the HTTP response when displaying a traceback, so should mention that here.